index.html

```diff
@@ -1,216 +1,243 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Moses' Portfolio</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Anton&family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
-    <header>
-            <h2 class="logo"><a href="#main-section">Moses</a></h2>
-            <nav class="nav-link">
-                <ul>
-                    <li class="trans"><a href="#portfolio-works">Portfolio</a></li>
-                    <li class="trans"><a href="#About">About</a></li>
-                    <li class="trans"><a href="#Contact">Contact</a></li>
-                </ul>
-            </nav>
-            <div class="menu">
-                <ul>
-                    <li><a href="#"><img src="image-source/Icon - Menu.png" alt="Humberger_menu"></a></li></ul>
-            </div>
-    </header>
-    <main>
-        <section id="main-section" class="main-section">
-            <h1>I’m Moses <br> Glad to see you!</h1>
-            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <h2 class="let">LET'S CONNECT</h2>
-                <ul class="nav-links">
-                    <li class="lift"><a href="https://twitter.com/Ezechukwuchibu2"><img src="image-source/twitter.svg" alt="twitter"></a></li>
-                    <li class="lift"><a href="https://www.linkedin.com/in/ezechukwu-chibuike/"><img src="image-source/Linkedin icon.svg" alt="Linkedin"></a></li>
-                    <li class="lift"><a href="#"><img src="image-source/medium.svg" alt="Medium"></a></li>
-                    <li class="lift"><a href="https://github.com/Moses-chibuike"><img src="image-source/github.svg" alt="github"></a></li>
-                    <li class="lift"><a href="#"><img src="image-source/angellist.svg" alt="angellist"></a></li>
-                </ul>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>moses Portfolio Homepage</title>
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/a2da67af29.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div class="modal">
+      <button class="cancels" type="button">&times;</button>
```

Comment:
Although you have done the changes requested by the previous CR there is a new problem that happened in design which is a project requirement. 

Your Popup is different in styles in comparison with the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A418&t=VLfEAsQms5aJygmA-0). In the template, the popup comes on almost the complete page but with some margins but in your case, it's taking the height as auto. Please consider making the design exactly to the template it will make your portfolio looks more professional. I am attaching screenshots of the comparison so you understand it well.

## Your Design: 

![image](https://user-images.githubusercontent.com/94223367/216107498-6a747fe3-3a3b-4fb9-8278-34bff5a350ce.png)

## Figma Design:
![image](https://user-images.githubusercontent.com/94223367/216107573-1fa87d4c-2fad-4473-b113-3e9e1ba0b579.png)

Reply:
Thanks for the review @ ABDULALI3468